### PR TITLE
fix(DomElementSchemaRegistry): add time element schema

### PR DESCRIPTION
**Pull request by lacolaco**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)
#11219

**What is the new behavior?**

Recognized correctly

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: modules/@angular/compiler/test/schema/schema_extractor.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 const SVG_PREFIX = ':svg:';
 const HTMLELEMENT_NAMES =
-    'abbr,address,article,aside,b,bdi,bdo,cite,code,dd,dfn,dt,em,figcaption,figure,footer,header,i,kbd,main,mark,nav,noscript,rb,rp,rt,rtc,ruby,s,samp,section,small,strong,sub,sup,u,var,wbr';
+    'abbr,address,article,aside,b,bdi,bdo,cite,code,dd,dfn,dt,em,figcaption,figure,footer,header,i,kbd,main,mark,nav,noscript,rb,rp,rt,rtc,ruby,s,samp,section,small,strong,sub,sup,time,u,var,wbr';
```
Comment (vicb) [inline modules/@angular/compiler/test/schema/schema_extractor.ts]: I know this code is a little bit confusing and I need to do some docs/refactoring.

`HTMLELEMENT_NAMES` are for `HtmlElement` interface.

`<time>` is `HtmlTimeElement`

could you please update your PR

File: modules/@angular/compiler/src/schema/dom_element_schema_registry.ts [hunk 2]
```diff
@@ -218,6 +218,8 @@ const SCHEMA: string[] = ([
   ':svg:title^:svg:|',
   ':svg:use^:svg:graphics|',
   ':svg:view^:svg:|#zoomAndPan',
+  // TODO: should be auto-generated https://github.com/angular/angular/issues/11219
```
Comment (vicb) [inline modules/@angular/compiler/src/schema/dom_element_schema_registry.ts]: TODO in this PR :)

**Comment by vicb** [comment]
@laco0416 taking this PR over

**Comment by trumbitta** [comment]
@vicb has this been merged and released? I'm having issues using the `time` element in a template with Angular 2.4

**Comment by DzmitryShylovich** [comment]
@trumbitta what kind of issues? http://plnkr.co/edit/hqiAtdLqWKOhhaZwxbYk?p=preview

**Comment by trumbitta** [comment]
I see, no issues in the plunkr... but in my app Angular says it doesn't know the `datetime` attribute of the `time` element... I guess I'll have to double-check my code

**Pull request closed without merging**